docs/platform-feature-parity.md

```diff
@@ -0,0 +1,171 @@
+# Platform feature-parity
+
+macOS and Windows binaries are new and considered alpha-quality. Some functionality may be missing, some may be broken. (Enhancements and bug-reports welcome, please see [#551: Multi-OS support](https://github.com/aelsabbahy/goss/issues/551))
+
+This matrix attempts to track parity across platforms.
+
+## How to use this doc
+
+### Legend
+
+* `x`    - full supported/tested
+* `w`    - full working/tested, community support
+* `wp-pt` - works partially / partially tested
+  * this is ambiguous; where you see this, check into the test coverage within `integration-tests/goss/{darwin|windows}/{test}.goss.yaml` for more detail. It might be that not all features work as on `linux`, it might be that not all features are covered by automated tests.
+* `w-nt` - works no automated tests
+* `ni`   - not implemented / needs implementation
+* `n/a`  - Not applicable for this platform
+* `#585` - Link to bug(s) for implementation, fixing
+* ` `    - (blank) - not yet tried, no data
+
+### Contributing
+
+The current integration test approach is only appropriate for validating `linux` binaries against `linux` OS/arch combinations.
+
+Validating `macOS` and `Windows` binaries requires adding coverage that runs on those platforms within Travis, but since Travis does not support containerised builds for either platform, assertions are limited to assert against the state of the CI hosts, where we're relying on that to predictable.
+
+You can find goss-files that are used to populate this matrix within `integration-tests/goss/{darwin|windows}/{test}.goss.yaml`. Where a feature does note work the same as linux, it is commented. The intent is to end up with a set of running-and-passing tests.
+
+## Matrix - tests/assertions
+
+| Test                | `linux` | `macOS` | `Windows` |
```

Comment:
Skimming though this on my phone looks like macos has a lot more testing than this chart let's on?

Reply:
Yeah, updating that shortly. Wanted to see it in CI before filling it in, since I'm actually using CI to iterate that now I've got the groundwork in place.